src/utils/seed.ts

```
@@ -0,0 +1,678 @@
+      transactionId: `txn_${faker.string.alphanumeric(24)}`,
+      paymentStatus: paymentStatuses[Math.floor(Math.random() * paymentStatuses.length)],
+      refunded: 0
+    });
+  }
+}
+
+
+async function main() {
+  try {
+    await seed();
+    console.log('Seeding completed');
```

ESLint

[error] Unexpected console statement.

(no-console)